ansible/roles_ocp_workloads/ocp4_workload_example/readme.adoc

```diff
@@ -22,25 +22,23 @@
 *** This role doesn't do anything here
 *** Debug task will print out: `remove_workload Tasks completed successfully.`
 
-== Review the defaults variable file
+== The defaults variable file
 
 * This file link:./defaults/main.yml[./defaults/main.yml] contains all the variables you need to define to control the deployment of your workload.
 * The variable *ocp_username* is mandatory to assign the workload to the correct OpenShift user.
```

Comment:
Want to add the same comment as in the other file that this will be overridden by CloudForms?